### ISSUE TITLE
cmake - I3_USE_ROOT macro set w/o root installed (Trac #1131)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1125]
if USE_ROOT is given to cmake, the C macro I3_USE_ROOT is set even if root isn't found.

see #1115

Migrated from https://code.icecube.wisc.edu/ticket/1131
```json
{
    "status": "closed", 
    "changetime": "2019-01-11T23:59:15", 
    "description": "if USE_ROOT is given to cmake, the C macro I3_USE_ROOT is set even if root isn't found.\n\nsee #1121", 
    "reporter": "nega", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1547251155701656", 
    "component": "cmake", 
    "summary": "cmake - I3_USE_ROOT macro set w/o root installed", 
    "priority": "blocker", 
    "keywords": "", 
    "time": "2015-08-17T18:43:53", 
    "milestone": "", 
    "owner": "nega", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150817T18:44:52: nega changed description from:

> if USE_ROOT is given to cmake, the C macro I3_USE_ROOT is set even if root isn't found.

to:

> if USE_ROOT is given to cmake, the C macro I3_USE_ROOT is set even if root isn't found.
> 
> see #1121

**Comment by icecube-trac** [comment]
Trac update at 20150820T16:06:33: 
* david.schultz changed status from "new" to "assigned"
* david.schultz commented:

> raising priority because this broke my build


* david.schultz changed priority from "normal" to "blocker"
* david.schultz changed owner from "" to "nega"

**Comment by icecube-trac** [comment]
Trac update at 20150820T16:41:37: 
* david.schultz commented:

> In changeset 2232:
> 
> ```text
> #!CommitTicketReference repository="IceTray" revision="2232"
> stop using USE_ROOT to test things. should fix #1125
> ```


* david.schultz changed status from "assigned" to "closed"
* david.schultz changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20190111T23:59:15: nwhitehorn commented:

> In changeset 170095:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="170095"
> stop using USE_ROOT to test things. should fix #1125
> ```
